<?xml version="1.0"?>
<robot name="igus_rebel" >

  <link name="world" />

  <link name="base_link">
    <visual>
      <geometry>
        <mesh filename="package://igus_rebel_description/meshes/robolink_rebel/Robolink_RebeL_000.dae"/>
      </geometry>
    </visual>
    <collision>
      <geometry>
        <mesh filename="package://igus_rebel_description/meshes/robolink_rebel/Robolink_RebeL_000.dae"/>
      </geometry>
    </collision>
    <inertial>
      <mass value="1.0" />
      <inertia ixx="1.0" iyy="1.0" izz="1.0" ixy="0.0" ixz="0.0" iyz="0.0" />
    </inertial>
  </link>

  <joint name="world_joint" type="fixed">
    <parent link="world" />
    <child link = "base_link" />
    <origin xyz="0.0 0.0 0.0" rpy="0.0 0.0 0.0" />
  </joint>


  <link name="link_1">
    <visual>
      <geometry>
        <mesh filename="package://igus_rebel_description/meshes/robolink_rebel/Robolink_RebeL_001.dae"/>
      </geometry>
      <origin rpy="0 0 3.1416" xyz="0.0 0.0 -0.0537"/>
    </visual>
    <collision>
      <geometry>
        <mesh filename="package://igus_rebel_description/meshes/robolink_rebel/Robolink_RebeL_001.dae"/>
      </geometry>
      <origin rpy="0 0 3.1416" xyz="0.0 0.0 -0.0537"/>
    </collision>
    <inertial>
      <mass value="0.1" />
      <inertia ixx="0.03" iyy="0.03" izz="0.03" ixy="0.0" ixz="0.0" iyz="0.0" />
    </inertial>
  </link>

  
  <joint name="joint1" type="revolute">
    <parent link="base_link"/>
    <child link="link_1"/>
    <origin rpy="0.0 0.0 3.1416" xyz="0.0 0 0.126"/>
    <axis xyz="0 0 -1"/>
    <limit effort="330.0" lower="-3.12414" upper="3.12414" velocity="2.16"/> <!-- limit taken from the controller -->
    <dynamics damping="0.0" friction="0.0"/>
  </joint>

</robot>

